packages/cdktf-cli/bin/cmds/helper/init.ts

```diff
@@ -141,15 +141,22 @@ This means that your Terraform state file will be stored locally on disk in a fi
         fs.writeFileSync(path.resolve(destination, "main.ts"), code, "utf8");
         fs.writeFileSync(
           path.resolve(destination, "cdktf.json"),
-          cdktfJson,
+          JSON.stringify(cdktfJson, null, 2),
           "utf8"
         );
 
+        const { terraformModules, terraformProviders } = cdktfJson as {
+          terraformModules: string[];
```

Comment:
Instead of using `as` you could also add proper types to the result from `convertProject` as this is not a string anymore now.